src/visualizer/objectFunctions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @description This file contains functions that use objects, and returns objectrelated stuff
+ */
+
+(function(){
+    window.objectFunctions = {getLSValuesOfGroupNameAsKey, getstudentByStudentName, getStudentIdxInGroupByStudentName, getLSValuesOfGroup};
+    /**
+     * @summary Creates and returns a new array with the learningstyle values of the learningstyle with the learningstyle name provided in the learnStyleName argument
+     * @param {object} group the group from which to get the data
+     * @param {string} learnStyleName the name of the learningstyle e.g. "activeReflective"
+     * @returns {object} returns an object with the students of the groups values in the given learningstyle, in the format {studentname: learningstylevalue} e.g. {jens: -3}
+     */
+    function getLSValuesOfGroupNameAsKey(group, learnStyleName){
+        const resArray = new Array();
+        for (const student of group.students) {
+            resArray[student.name] = student.criteria.learningStyles[learnStyleName];
+        }
+        return resArray;
+    }
+
+    /**
+     * @summary Gets and return the student object from the studentName, return the student object where (student.name === studentName)
+     * @param {object} group the group from which to find and get the student
+     * @param {string} studentName the name of the student to find and return
+     * @returns {object} return the student object from the group object where student.name === studentName
+     */
+    function getstudentByStudentName(group, studentName){
```

Comment:
Hvorfor fuck i satans vilde helvede har i nogen sinde brug for denne funktion?

Reply:
Øh... misclick, fixed, findes ikke længere XD